portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,20 @@ function addRandomFunFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+var currentProjIndex = 0;
+var projects = ["images/proj1.png", "images/proj2.png", "images/proj3.png", "images/proj4.png"];
+var changeProjectTimeMs = 3000;
```

Comment:
The page looks great! I think this timeout is a little short though; I wasn't able to read through any project's description before the next one popped in. Consider either raising this to 8000, listing all the projects out at once, or allowing the user to navigate between them manually.